daemon/event.go

```diff
@@ -111,6 +127,34 @@ func (d *daemon) CreateEvent(req *pb.CreateEventRequest, resp pb.Daemon_CreateEv
 		return err
 	}
 	d.startEvent(ev)
+
+	return nil
+}
+
+func (d *daemon) bookEvent(ctx context.Context, req *pb.CreateEventRequest) error {
```

Comment:
Do we have to create a method in the webclient for this function right?

Reply:
No, I do not think, we have to.